### DISCUSSION TITLE
Can I Change Origin of Img in BB Code Text?

### DISCUSSION THREAD

**Discussion by rtlqwm324** [rexrainbow/phaser3-rex-notes#342] [Q&A] [answered]
Hello Rex, I hope this message finds you well. I've been using your plugins and they've been working great for me. I'm looking to add an img button to my BB CODE TEXT and apply an animation that changes the size of the image depending on mouse input. I've been using addImage to adjust the size, but when I run updateText, the origin of the image is always at 0,0, which causes it to expand only to the bottom-right. Is there a way to change the origin of the img to 0.5,0.5?

**Comment by rexrainbow** [comment, marked as answer]
Do you mean that, image pasted on BBCodeText by `[img=imgKey]` tag? It does not have origin setting (yet). Can try to add this feature later.
